docker-compose.yml

```diff
@@ -1,44 +1,36 @@
-version: '1.0'
+version: '1.2'
 
 services:
   db:
     image: postgres:latest
-    restart: always
-    ports:
-      - "8088:8088"
+    #restart: always
     volumes:
-      - data:/var/lib/postgresql/data
+      - ./data/db:/var/lib/postgresql/data
```

Comment:
Können wir statt einem bind mount auch ein "normales" volume nehmen? Brauchen wir den Zugriff auf den Ordner vom Hostsystem?

Falls wir das auf dem Host brauchen ist es wahrscheinlich `data/` in die `.gitignore` zu packen.

Reply:
um ehrlich zu sein keine ahnung ^^ habs erstmal in die gitignore mit aufgenommen. ob der bind notwendig ist bzw was fuer vorteile das hat muesste ich nochmal recherchieren, ausser du hast ne ahnung?